dotcom-rendering/scripts/webpack/webpack.config.server.js

```diff
@@ -19,6 +32,8 @@ module.exports = ({ sessionId }) => ({
 	},
 	externals: [
 		'@loadable/component',
+		'express',
+		'log4js',
 		require('webpack-node-externals')({
```

Comment:
this _should_ be excluding everything in `node_modules` already – if this change works it suggests it is not? 

[the docs](https://www.npmjs.com/package/webpack-node-externals) (for v3) suggest webpack 5 config should use `externalsPresets` instead of `target`, i wonder if updating `webpack-node-externals` v3 and changing that config setting will get it working again?

Reply:
I shall investigate and update here with what I find :)

Reply:
Now resolved! PR Description updated with more info!